docs/DeveloperGuide.md

```diff
@@ -975,7 +975,7 @@ Upon identification of such invalid `NAME` field values, PatientSync should then
 ### 4.7 Implementation on PatientHospitalId
 
 Presently, the patient's hospital ID referred to as `PATIENT_HOSPITAL_ID`, is implemented as String.
-It currently uses regex expression `^[0-9]+$` to validate the input for patient's hospital ID. 
+It currently uses regex expression `^[0-9]+$` where it is a String that consists only numeric characters to validate the input for patient's hospital ID. 
```

Comment:
Similar to the previous comment, the word **it** is quite ambiguous! Perhaps could consider rephrasing to: PatientSync  currently uses regex expression `^[0-9]+$` to check that the user input for `PATIENT_HOSPITAL_ID` is a String that consists only numeric characters to validate the input for patient's hospital ID.

Reply:
Agree!